packages/syft/src/syft/node/node.py

```diff
@@ -843,6 +843,19 @@ def forward_message(
                 signed_result = client.connection.make_call(api_call)
                 result = debox_signed_syftapicall_response(signed_result=signed_result)
 
+                # relative
+                from ..store.blob_storage import BlobRetrievalByURL
+
+                # In the case of blob storage, the gateway downloads the result and then passes it to
```

Comment:
nitpick: Can we add like a TODO comment, that this needs a longer fix.

Reply:
Done [87c2676](https://github.com/OpenMined/PySyft/pull/8290/commits/87c2676d16584d461543499a0debbaf2ea51ff70)